### ISSUE TITLE
several script errors

### ISSUE THREAD

**Issue by adamsewell** [Jay-Rad/Remotely#62]
1) 

> Error while running nginx -c /etc/nginx/nginx.conf -t.
> nginx: [emerg] server name "*." is invalid in /etc/nginx/sites-enabled/remotely:3
> nginx: configuration file /etc/nginx/nginx.conf test failed
>The nginx plugin is not working; there may be problems with your existing configuration.
>The error was: MisconfigurationError('Error while running nginx -c /etc/nginx/nginx.conf -t.\n\nnginx: [emerg] server name "*." is invalid in /etc/nginx/sites-enabled/remotely:3\nnginx: configuration file /etc/nginx/nginx.conf test failed\n',)

2) 
> --2019-12-12 21:06:58--  https://remotely.lucency.co/Downloads/linux-x64/Server.zip
> Resolving remotely.lucency.co (remotely.lucency.co)... 23.94.74.198
> Connecting to remotely.lucency.co (remotely.lucency.co)|23.94.74.198|:443... connected.
> HTTP request sent, awaiting response... 404 Not Found
> 2019-12-12 21:07:00 ERROR 404: Not Found.
> unzip:  cannot find or open Server.zip, Server.zip.zip or Server.zip.ZIP.
> rm: cannot remove 'Server.zip': No such file or directory

**Comment by Jay-Rad** [comment]
The short story is: that's an old version of the script.

Here's the longer story.  I'm no longer hosting a server package.  Not because I don't want to, but because the build process went through a lot of changes while I was trying to mitigate false virus detections.

Ultimately, I had to provide the desktop client in a ClickOnce installer, and that requires the server address to be compiled into them.  So I couldn't offer a generic package anymore.  Each client has to be built specifically for the server it talks to.

Of course, I could probably contact the antivirus companies and try to get my software whitelisted.  They'd probably require that I use a code signing cert, though, and I don't have money to spare right now.

So for now, that's how it is.  In the future, I'd like to make the desktop client a single-file EXE again.  Maybe some day when I have more time to chase down AV companies and spare change to throw at a cert.

As for any other errors...  I just ran the most recent script on Ubuntu 18.04, and the server fired up without issues.
